src/xrGame/configs_dump_verifyer.h

```diff
@@ -21,8 +21,7 @@ class configs_verifyer
     bool const verify(u8* data, u32 data_size, string256& diff);
 
 private:
-    typedef u8 sha_checksum_t[crypto::xr_sha256::digest_length];
-    bool const verify_dsign(u8* data, u32 data_size, sha_checksum_t& sha_checksum);
+    bool const verify_dsign(u8* data, u32 data_size, crypto::xr_sha1::hash_t& sha_checksum);
```

Comment:
u8* -> const u8* const data
Const, for the god of const!

Reply:
Will do

Reply:
What is the point of returning a *const pointer to a const value* instead of just a *pointer to a const value*?

Reply:
@nitrocaster, `data` var should not be changed in `verify_dsign`, so `data` should be a pointer to const.

Reply:
I decided not to change this part of code, because it is from another project.